lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
 
+    //Const for main stylesheet.
     const stylesheet = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
-    stylesheet.rel = 'stylesheet';
     stylesheet.href = 'chrome://universalsearch-skin/content/style.css';
+    stylesheet.rel = 'stylesheet';
     stylesheet.type = 'text/css';
     stylesheet.id = 'universal-search-stylesheet';
     stylesheet.style.display = 'none';
 
+    //Const for platform-specific stylesheet.
+    const stylesheetPlatform = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
+    stylesheetPlatform.rel = 'stylesheet';
+    stylesheetPlatform.type = 'text/css';
+    stylesheetPlatform.id = 'universal-search-stylesheetPlatform';
+    stylesheetPlatform.style.display = 'none';
+
+    // If OS is Mac, load style-osx.css.
+    if (osString == 'Darwin') {
+      stylesheetPlatform.href = 'chrome://universalsearch-skin/content/style-osx.css';
+    //If OS is Linux, load style-linux.css.
+    } else if (osString == 'Linux') {
+      stylesheetPlatform.href = 'chrome://universalsearch-skin/content/style-linux.css';
+    //Otherwise, load style-windows.css.
+    } else {
```

Comment:
I think it'd be better to make linux the `else` case, so, explicitly check for Darwin and WINNT, and the else will load the linux styles.

The reason is that there are [many odd fringe values](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Build_Instructions/OS_TARGET) of Services.appinfo.OS, most of which are Linux-y (BSDs and so on). I think the windowing system on those fringe OSes is probably closest to Linux, so it's the better fallback choice.